ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx

```diff
@@ -93,4 +109,32 @@ export class CronWorkflowDetails extends BasePage<RouteComponentProps<any>, Stat
                 document.location.href = uiUrl('cron-workflows');
             });
     }
+
+    private suspendCronWorkflow() {
+        const wf = {...this.state.cronWorkflow};
+        wf.spec.suspend = true;
+        services.cronWorkflows
+            .update(wf, this.name, this.namespace)
+            .catch(e => {
+                this.appContext.apis.notifications.show({
+                    content: 'Failed to suspend cron workflow ' + e,
+                    type: NotificationType.Error
+                });
+            })
+            .then(() => this.getCronWorkflow());
```

Comment:
Doesn't the `update` function return the new, updated Workflow? I don't think we need to make this extra API call

Reply:
You're right. I've updated this part. Thanks!